src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import cx from 'classnames';
+
+export const ACCORDIONPANELGROUP_CLASS = 'accordionPanelGroup';
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			clickedPanel: null
+		};
+
+		this.clonePanel = this.clonePanel.bind(this);
+		this.setClickedPanel = this.setClickedPanel.bind(this);
+
+		this.accordionPanels = this.props.accordionPanels.map(this.clonePanel);
+	}
+
+	/**
+	 * @param clickedPanel is the `AccordionPanel` component passed in from `_handleToggle`
+	 * @returns {undefined}
+	 *
+	 * Keeps track of the clicked panel in order to support AccordionPanelGroups that only
+	 * have one panel open at a time.
+	 */
+	setClickedPanel(clickedPanel) {
+		if (!this.props.multiSelectable) {
+			this.setState({ clickedPanel });
+		}
+	}
+
+	/**
+	 * @param {Object} accordionPanel - `AccordionPanel` components to clone
+	 * @param {number} i - index of the `AccordionPanel`
+	 * @param {boolean} isOpen - whether the `AccordionPanel` is open or not
+	 * @returns {Array} `AccordionPanel` components with props from `AccordionPanelGroup`
+	 */
+	clonePanel(accordionPanel, i, isOpen) {
+		const panelProps = {
+			key: i,
```

Comment:
trivial, but feel free to name the index argument `key` so that you don't have to alias it.